scripts/generateOpenApi.ts

```diff
@@ -13,10 +13,10 @@ async function run() {
   const app = await getApp({
     healthchecksEnabled: false,
     monitoringEnabled: false,
-    arePeriodicJobsEnabled: false,
-    amqpConsumersEnabled: false,
-    enqueuedJobsEnabled: false,
-    enqueuedJobQueuesEnabled: false,
+    periodicJobsEnabled: false,
+    messageQueueConsumersEnabled: false,
+    jobWorkersEnabled: false,
```

Comment:
in a hindsight I think `enqueuedJobs` naming is still better and made more sense, will change in the lib too